app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -24,16 +24,20 @@ public function scheduleFeed($schedule) {
      */
     public function createFeed($fileFormat)
     {
-        Mage::log($fileFormat . ": "  . __METHOD__ . " cron", null, 'cron.log');
+        if (!isset($fileFormat) || trim($fileFormat)==='' ) {
```

Comment:
I agree with you on this issue but regarding backwards compatibility (in the previous versions cron config calls cron::createFeed() function) it's necessary to set the default value within createFeed() function. Anyway I am open for suggestions if you have another idea how to solve this.